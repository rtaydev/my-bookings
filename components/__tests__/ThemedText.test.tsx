import React from "react";
import renderer from "react-test-renderer";
import { ThemedText } from "../ThemedText";

it("renders correctly", () => {
  const tree = renderer
    .create(<ThemedText>Snapshot test!</ThemedText>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("applies correct styles based on type", () => {
  const tree = renderer
    .create(<ThemedText type="title">Title</ThemedText>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
