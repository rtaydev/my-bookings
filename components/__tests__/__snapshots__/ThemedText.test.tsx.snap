// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`applies correct styles based on type 1`] = `
<Text
  style={
    [
      {
        "color": "#000",
      },
      undefined,
      {
        "fontSize": 32,
        "fontWeight": "bold",
        "lineHeight": 32,
      },
      undefined,
      undefined,
      undefined,
      undefined,
    ]
  }
>
  Title
</Text>
`;

exports[`renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#000",
      },
      {
        "fontSize": 16,
        "lineHeight": 24,
      },
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
    ]
  }
>
  Snapshot test!
</Text>
`;
